examples/oneapi/cpp/source/decision_forest/df_reg_dense_batch.cpp

```diff
@@ -18,45 +18,47 @@
 #include "oneapi/dal/algo/decision_forest.hpp"
 #include "oneapi/dal/io/csv.hpp"
 
-using namespace oneapi;
+namespace onedal = oneapi::dal;
```

Comment:
Is it xLib suggesting to have `onedal` alias for `oneapi::dal`?

Reply:
No - they suggest more general things:
1. Do not put `using namespace` in the examples
2. Create and aliases instead

Reply:
Change onedal alias to dal